carma-messenger-core/cpp_message/src/MobilityPath_Message.cpp

```diff
@@ -151,7 +151,7 @@ namespace cpp_message
         cav_msgs::MobilityHeader Header;
         Header=plainMessage.header;
         Mobility_Header encode_header;
-        MobilityHeader_t* header=encode_header.toASN1_mobility_header_message(Header);
+        MobilityHeader_t* header=(encode_header.toASN1_mobility_header_message(Header)).get();
```

Comment:
Need to check if the conversion succeeded.

Reply:
Updated